src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -341,6 +341,20 @@ public string GetHelpMaml(string helpItemName, string path)
         #endregion
 
         #region CmdletProvider members
+        // OneDrive placeholder support
+#if !UNIX
```

Comment:
In this case it is very easy to fix the Code Factor violation "Field names must not contain underscores."
Same file `FileSystemProvider.cs` already has `private static class NativeMethods`, so please move this new P/Invoke things there. Thank you.

Here is the guidance from Code Factor:

> If the field or variable name is intended to match the name of an item associated with Win32 or COM, and thus needs to include underscores, place the field or variable within a special NativeMethods class. A NativeMethods class is any class which contains a name ending in NativeMethods, and is intended as a placeholder for Win32 or COM wrappers. StyleCop will ignore this violation if the item is placed within a NativeMethods class.

Reply:
We already have some NativeMethods classes and could consider reusing one.